rever/activities/push_tag.xsh

```diff
@@ -31,6 +32,12 @@ class PushTag(Activity):
                          desc="Tags the current version.")
 
     def _func(self, remote=None, target=None):
+        if remote is None:
+            remote = ${...}.get('TAG_REMOTE', None)
+            warnings.warn(DeprecationWarning('TAG_REMOTE has been deprecated '
```

Comment:
This warning is issued even if TAG_REMOTE doesn't exist.

Reply:
Yes, that was silly sorry.